storybookjs/storybook
# add watch mode for build-storybook

Issue: https://github.com/storybooks/storybook/issues/2762

## What I did

Added `-w --watch` option for `build-storybook`.

## How to test

Link storybook to an existing app, run `build-storybook --watch`. Watch should be enabled.

Is this testable with jest or storyshots?

No.

Does this need a new example in the kitchen sink apps?

Not sure / don't think so.

Does this need an update to the documentation?

Yes

If your answer is yes to any of these, please make sure to include it in your PR.
